### ISSUE TITLE
How do I specify transaction isolation level when using newSuspendedTransaction?

### ISSUE THREAD

**Issue by gps** [JetBrains/Exposed#981]
I'm trying to specify a `transactionIsolation` level for a transaction using `newSuspendedTransaction`. I don't see it taking a parameter for `transactionIsolation`?

The non coroutine version `transaction` does:

```kotlin
    fun <T> foo(block: () -> T): T {
        return transaction(
            db = db,
            transactionIsolation = Connection.TRANSACTION_SERIALIZABLE,
            repetitionAttempts = 1
        ) {
            block()
        }
    }
```

Is there a way to do this with `newSuspendedTransaction` as well?

Thanks!

**Comment by Tapac** [comment]
@gps should be available in 0.26.1, please check.

**Comment by gps** [comment]
> @gps should be available in 0.26.1, please check.

@Tapac Can confirm that it is in 0.26.1. Thanks!
